powerquery-docs/Connectors/DatabricksCloud.md

```diff
@@ -0,0 +1,80 @@
+
+1. In the Get Data experience, search for **databricks** to shortlist the two Databrick connectors: "Azure Databricks" and "Databricks". Choose "Azure Databricks" if your Databricks runs in the Azure cloud; for AWS or GCP, choose "Databricks". (If the entry for "Databricks" is marked with a "(Beta)" suffix, contact your Databricks representative for guidance on using the connector.)
+
+    ![Get Data from Databricks Cloud.](./media/databricks-cloud/get-data-dbc.png)
+
+2. You will need to provide the "Server hostname" and "HTTP Path" for your Databricks SQL Warehouse. Refer [here](https://docs.microsoft.com/en-us/azure/databricks/integrations/bi/jdbc-odbc-bi#get-server-hostname-port-http-path-and-jdbc-url) for instructions to look up your "Server hostname" and "HTTP Path". Enter these accordingly. You may optionally supply a default catalog and/or database under **Advanced options**. Select **OK** to continue.
+
+    ![Specify your Databricks SQL Warehouse.](./media/databricks-cloud/azdbc-sql-endpoint.png)
+
+3. You will need provide your credentials to authenticate with your Databricks SQL Warehouse. For Azure Databricks, you can choose between 1) basic user/password; 2) Personal Access Token; and 3) Azure Active Directory (AAD). For AWS or GCP, your choice is between 1) basic user/password; 2) Personal Access Token; and 3) OAuth. Refer [here](https://docs.microsoft.com/en-us/azure/databricks/sql/user/security/personal-access-tokens) for instructions on generating a Personal Access Token (PAT). For organizational account (when using Azure Databricks), you will be prompted to log into your Azure Databricks cluster. For OAuth (when using Databricks on AWS or GCP), you will be prompted to log into your Databricks SQL Warehouse.
+
+    ![Specify an appropriate token.](./media/databricks-cloud/azdbc-pat.png)
+
+    > [!NOTE]
+    > Once you enter your credentials for a particular **Databricks** cluster, Power BI Desktop caches and reuses those same credentials in subsequent connection attempts. You can modify those credentials by going to **File > Options and settings > Data source settings**. More information: [Change the authentication method](../ConnectorAuthentication.md#change-the-authentication-method)
+
+4. Once you successfully connect, the **Navigator** window shows the data available to you on the cluster. You can choose to either **Transform Data** to transform the data using Power Query or **Load** to load the data in Power Query Desktop. 
+
+    ![Image of Power Query navigator loading Databricks Cloud data to desktop app.](./media/databricks-cloud/navigator-with-filter.jpeg)
+
+## Connect to Databricks Cloud data from Power Query Online
+
+To connect to Databricks Cloud from Power Query Online, take the following steps:
+
+1. In the Get Data experience, select the **Dataflow** category. (Refer [here](https://docs.microsoft.com/en-us/power-bi/transform-model/dataflows/dataflows-create) for instructions.)
```

Comment:
Dataflow is a new workflow which will be supported soon. However majority of the customers are using PowerBI Desktop with PowerBI Cloud (service or gateway). I think we should also have a section for PowerBI (service, gateway).

Reply:
I use the language here ("Power Query Online") to be consistent with the other connectors. Microsoft communicates the Power BI "brand" as different manifestations of "Power Query".